docs/sources/reference/commandline/cli.md

```diff
@@ -1635,6 +1636,9 @@ timestamp, for example `2014-09-16T06:17:46.000000000Z`, to each
 log entry. To ensure that the timestamps for are aligned the
 nano-second part of the timestamp will be padded with zero when necessary.
 
+The `docker logs --since` argument shows logs of a container generated only
+since the specificed date. It can also be used with `--follow` and `--tail`.
```

Comment:
s/specificed/specified/

date -> date/time or timestamp.

"It can also be used with" -> "The `--since` can be combined with the `--follow` and `--tail` options."